docs/topics/Gateway.md

```diff
@@ -563,10 +563,30 @@ A user's presence is their current state on a guild. This event is sent when a u
 |-------|------|-------------|
 | user | [user](#DOCS_USER/user-object) object | the user presence is being updated for |
 | roles | array of snowflakes | roles this user is in |
-| game | object | null, or an object containing one key of "name" |
+| game | ?[game](#DOCS_GATEWAY/game-object) object | the game being played/streamed, or null |
```

Comment:
`an addition to the users status, generally showing their currently "activity"` or smth along those lines